docs/codeql/ql-language-reference/annotations.rst

```diff
@@ -97,6 +97,20 @@ own body, or they must inherit from another class that overrides ``isSource``:
       // doesn't need to override `isSource`, because it inherits it from ConfigA
     }
 
+.. index:: additional
+.. _additional:
+
+``additional``
+==============
+
+**Available for**: |classes|, |algebraic datatypes|, |type unions|, |non-member predicates|, |modules|, |aliases|, |signatures|
+
+The ``additional`` annotation can be used on declarations directly inside of modules that implement |module signatures|.
```

Comment:
What does `directly` in `declarations directly inside of modules` mean here? I assume it relates to nested modules. 

Can't we simplify it to something along the lines of:
```
The `additional` annotation can be used on declarations in modules. All declarations in modules with |module signatures| that are not required by a module signature must be annotated with `additional`.
```